### PR TITLE
Update UG for CS2101

### PR DESCRIPTION
**Pull request by JoelHo**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -58,32 +58,38 @@ Please note the following symbols used in the User Guide which may serve as poin
 **:information_source: Notes about the command format:**<br>
 
 * Words in `UPPER_CASE` are the parameters to be supplied by the user.<br>
-  e.g. in `add n/NAME`, `NAME` is a parameter which can be used as `add n/John Doe`.
+  e.g., in `add n/NAME`, `NAME` is a parameter that can be used as `add n/John Doe`.
```
Comment (chewterence) [inline docs/UserGuide.md]: is the comma in the front of this sentence intentional?

File: docs/UserGuide.md [hunk 2]
```diff
@@ -58,32 +58,38 @@ Please note the following symbols used in the User Guide which may serve as poin
...
 
 * Items in square brackets are optional.<br>
-  e.g `n/NAME [tag/TAG]` can be used as `n/John Doe tag/finance` or as `n/John Doe`.
+  e.g., `n/NAME [tag/TAG]` can be used as `n/John Doe tag/finance` or as `n/John Doe`.
 
 * Items with `…`​ after them can be used multiple times including zero times.<br>
-  e.g. `[tag/TAG]…​` can be used as ` ` (i.e. 0 times), `tag/finance`, `tag/finance tag/management` etc.
+  e.g.,`[tag/TAG]…​` can be used as ` ` (i.e. 0 times), `tag/finance`, `tag/finance tag/management` etc.
```
Comment (chewterence) [inline docs/UserGuide.md]: i noticed this same thing for the previous comment

File: docs/UserGuide.md [hunk 3]
```diff
@@ -58,32 +58,38 @@ Please note the following symbols used in the User Guide which may serve as poin
...
 
 * Parameters can be in any order.<br>
-  e.g. if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
+  e.g., if the command specifies `n/NAME p/PHONE_NUMBER`, `p/PHONE_NUMBER n/NAME` is also acceptable.
 
 * If a parameter is expected only once in the command but you specified it multiple times, only the last occurrence of the parameter will be taken.<br>
-  e.g. if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
+  e.g., if you specify `p/12341234 p/56785678`, only `p/56785678` will be taken.
```
Comment (chewterence) [inline docs/UserGuide.md]: perhaps we should use real phone numbers here

File: docs/UserGuide.md [hunk 4]
```diff
@@ -138,10 +144,10 @@ Adds a new passenger in the GME terminal.
 <div markdown="block" class="alert alert-info">
 
 **:information_source: Notes about the add command:**<br>
-* [TripDay](#6-glossary) is required to be a valid day of the week. e.g. `SUNDAY` or `FRIDAY`
-* [TripTime](#6-glossary) is required to be in the 24-hour format. e.g. `0530` or `2359` 
-* [Tag](#6-glossary) is required to be an alphanumeric input and can only have a maximum of 30 characters in length.
-* [Price](#6-glossary) is an optional amount that the passenger can include that describes how much they are willing to pay for a carpooling trip. It is required to be a positive whole number or a positive decimal number rounded to 2 decimal places.
+* [TripDay](#6-glossary) must be a valid day of the week. e.g., `SUNDAY` or `FRIDAY`
+* [TripTime](#6-glossary) must be in the 24-hour format. e.g., `0530` or `2359`
```
Comment (chewterence) [inline docs/UserGuide.md]: are the additional commas intentional?

File: docs/UserGuide.md [hunk 5]
```diff
@@ -190,16 +196,16 @@ Finds passengers whose names contain any of the given keywords.
...
+* Keywords separated by space will match all as a phrase rather than individual words.
+  e.g., `Hans Yang` will only return `Gruber Hans Yang` instead of `Bo Yang` and `Hans Gruber Yang`.
 * Prefixes for searching name `n/`, address `a/`, tag `tag/`, phone number `p/`, price `pr/`, day `d/` and time `t/`.
-* Searching with prefixes such as `n/` and `a/` which are anticipated to have multiple words separated by spaces will have multiple spaces shortened to one. Refer to 4th example for clarification.
+* Searching with prefixes such as `n/` and `a/` which are anticipated to have multiple words separated by spaces will have multiple spaces shortened to one. Refer to the 4th example for clarification.
 * `all` prefix for searching across names, addresses, tags and phone numbers quickly.
-* However, if any or all of the words are invalid arguments, searching with `all` will not tell you it is invalid. It will only show that no passengers with those arguments are found.
+	* However, if any or all of the words after the `all` prefix are invalid words, GME will not inform you as such. It will instead show that no passengers with those words are found.
```
Comment (chewterence) [inline docs/UserGuide.md]: this sentence could perhaps be rephrased.
"if any or all of the words after the `all` prefix are invalid words"
since the subject is already 'words', re-stating them again as 'invalid words' is not necessary.

then again there's the part 'any or all' which makes too many clauses in the sentence. perhaps could pick one of them, because 'any' or 'all' kind of imply the same thing, unless i may have gotten this wrong.

TLDR my suggestion is to remove the second word 'word', and remove the word all 'a' -> "if any of the words after the `all` prefix are invalid"

**Review by chewterence — COMMENTED** [review]
good changes, however please take a look at my comments.

**Review by ZechariahTan — APPROVED** [review]
LGTM
